src/models/stores/nav-tabs.ts

```diff
@@ -11,10 +11,10 @@ export const NavTabsConfigModel = types
   })
   .views(self => ({
     getNavTabSpec(tabId: ENavTab) {
-      return self.tabSpecs.find(tab => tabId === tab.tab);
+      return self.tabSpecs.find(tab => tabId === tab.tab && !tab.hidden);
```

Comment:
I'm not sure if this as a good idea or not. It seems reasonable to return a nav tab spec even if that spec is hidden.

I'm guessing that this change means that if an author hides a tab at the unit level, this tab will not be rendered in any of the problems even if that problem specifies this tab. That seems kind of reasonable however in most other cases the problem configuration can override the unit level config. 

From a data consistency point of view, it'd be better if the 'hide' action at the unit level would go through and remove the tab from all of the problems configs. But I get how that would make authoring annoying if you accidentally hide a tab at the unit level.

Reply:
I'm still working out with Leslie what this means - I think once I get the preview working in CLUE it will become obvious what to do but until then I didn't want to not hide them (if that makes sense)